crates/runtime/src/accelerated_table/refresh.rs

```diff
@@ -113,6 +113,7 @@ impl Refresher {
         self
     }
 
+    #[allow(clippy::too_many_lines)]
```

Comment:
is that possible to split this methods into multiple smaller ones?

I think we should try to avoid these outlier if no obvious reason.

Reply:
Updated: https://github.com/spiceai/spiceai/pull/1589/commits/64b8d3a6b434218e47cccc1b1fcc7024b4617aeb